.github/workflows/ci.yml

```diff
@@ -25,7 +25,7 @@ jobs:
         run: coveralls --service=github
     services:
       postgres:
-        image: postgres:11
+        image: postgres:14
```

Comment:
Just checking - does it work as of PostgreSQL 12? Since we plan to upgrade to 14, appropriate to test on 14.

Reply:
Yes, AFAIK this PR should work for version 12+, but I've been testing locally with 14.

There are some specific SQL/JSON path expressions that users might want to specify which require versions later than 12. The main example I've seen is datetime, which requires PostgreSQL 13.